### DISCUSSION TITLE
Add CORS headers to `https://registry.npmjs.org/`

### DISCUSSION THREAD

**Discussion by Jolg42** [npm/feedback#386] [Registry] [answered]
Originally created a while ago at https://github.com/npm/npm-registry-couchapp/issues/108

Adding CORS headers would make it possible to fetch registry data directly from browsers without proxy hacks.

**Comment by arcanis** [comment, marked as answer]
Duplicate of https://github.com/npm/feedback/discussions/117

**Reply by Jolg42** [reply]
Indeed (I forgot that one 😅) Thanks!
